api/core/tools/provider/builtin/xtuis/tools/xtuis_push.yaml

```
@@ -0,0 +1,40 @@
+identity:
+  name: xtuis_push
+  author: Jimmy Wu
+  label:
+    en_US: Send Message to Weixin/Wechat
+    zh_Hans: 虾推啥-微信推送
```

虾推啥 supports not only Wechat. Just use the name for this. And wechat support to be described in single function.
```suggestion
    zh_Hans: 虾推啥
```